src/pages/WishList/WishList.js

```diff
@@ -0,0 +1,35 @@
+import { useCart } from '../../contexts/CartProvider';
+import { WishListItemCard } from './WishListItemCart';
+import './WishList.css';
+
+export function WishList() {
```

Comment:
Could reduce the indentation space a bit.

Reply:
I have used tabs for indentation, so you can change the space per tab to make the code look however you like. In GitHub the indent size per tab is 8 by default, that's why it looks so spaced out.

Reply:
Just found that this default can't be changed, looks like I'll have to go back to using spaces for indentation.